_dashboards-assistant/index.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: default
+title: OpenSearch Assistant for OpenSearch Dashboards
+nav_order: 120
+has_children: false
+has_toc: false
+nav_exclude: true
+permalink: /dashboards-assistant/
+redirect_from:
+  - /dashboards-assistant/index/
+---
+
+This is an experimental feature and is not recommended for use in a production environment. For updates on the feature's progress or to leave feedback, go to the [`dashboards-assistant` repository](https://github.com/opensearch-project/dashboards-assistant) on GitHub or the associated [OpenSearch forum thread](https://forum.opensearch.org/t/feedback-opensearch-assistant/16741).
+{: .warning}
+
+For more information about ways to enable experimental features, see [Experimental feature flags]({{site.url}}{{site.baseurl}}/install-and-configure/configuring-opensearch/experimental/).
+{: .note}
+{: .warning}
+
+# OpenSearch Assistant for OpenSearch Dashboards
+Introduced 2.12
+{: .label .label-purple }
+
+The [OpenSearch Assistant toolkit](<insert-ML-cross-ref-link>) helps you create AI-powered assistants for OpenSearch Dashboards without requiring specialized query tools or skills.
+
+## Enabling OpenSearch Assistant
+
+To enable **OpenSearch Assistant** in OpenSearch Dashboards, locate your copy of the `opensearch_dashboards.yml` file and set the following option:
+```
+assistant.chat.enabled: true
+```
+{% include copy-curl.html %}
+Then configure the root `agent_id` through the following API:
+```
+PUT .plugins-ml-config/_doc/os_chat
+{
+    "type":"os_chat_root_agent",
+    "configuration":{
+        "agent_id": "your root agent id"
+    }
+}
+```
```

Comment:
We should call out this is a system index, so in security enabled domains, the user must be super admin to do this.
They can use this guide to make super admin calls: https://opensearch.org/docs/latest/security/configuration/system-indices/

Reply:
@lezzago Please review the following callout text to make sure I've captured your comment accurately:

_This example is a system index. In security-enabled domains, only super administrators have access to execute this code. For details about making super admin calls, see the [System indexes](https://opensearch.org/docs/latest/security/configuration/system-indices/) guide. For access permission, contact your IT administrator._